src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositives.java

```diff
@@ -32,62 +30,101 @@
 /**
  * Created by tsato on 12/28/16.
  */
-public class CountFalsePositives extends CommandLineProgram {
-    @Argument(doc = "a line-delimited list of mutect2 vcf files", fullName= "", shortName = "V", optional = false)
-    protected File vcfList;
 
-    @Argument(doc = "output file", fullName= "output", shortName = "O", optional = false)
-    protected File output;
+/** TODO: this class should be a variant walker. That way we can ask questions like how are the false positive rates
+ *  over these intervals and so on. For now, we look at the entire vcf and assume ICE intervals
+ */
+
+public class CountFalsePositives extends VariantWalker {
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "Output file (if not provided, defaults to STDOUT)", common = false, optional = true)
+    private File outputFile = null;
+
+    private PrintStream outputStream = null;
+
+    long targetTerriroty = 0;
+    long indelFalsePositiveCount = 0;
+    long snpFalsePositiveCount = 0;
+    String id;
+
+
 
     // TODO: eventually use tumor and normal sample names instead of the file name. To do so we must extract them from the vcf, which I don't know how.
     @Override
-    public Object doWork() {
-        Utils.regularReadableUserFile(vcfList);
-        try ( FalsePositiveTableWriter writer = new FalsePositiveTableWriter(output) ) {
-            List<String> vcfFilePaths = Files.readAllLines(Paths.get(vcfList.toString()));
-            List<FalsePositiveRecord> falsePositiveRecords = vcfFilePaths.stream().map(str -> new FalsePositiveRecord(new File(str))).collect(Collectors.toList());
-            writer.writeAllRecords(falsePositiveRecords);
+    public void onTraversalStart() {
```

Comment:
Yea but you can't tell which one is tumor or normal as we get a list of sample names (e.g. SM-612V3). I tried to glean this information from the "sample" header lines (as opposed to genotype sample names) but I wasn't successful there either. If it's alright with you I'm tempted to change genotype sample names back to tumor and normal.